ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -136,33 +88,33 @@ web:
     bindPoints:
       #interface - required
       # A host:port string on which network interface to listen on. 0.0.0.0 will listen on all interfaces
-      - interface: {{.Web.BindAddress}}:{{ .Web.Port }}
+      - interface: 0.0.0.0:{{ .ZitiEdgeCtrlPort }}
         # address - required
         # The public address that external incoming requests will be able to resolve. Used in request processing and
         # response content that requires full host:port/path addresses.
-        address: {{.Web.ExternalAddress}}:{{ .Web.ExternalPort }}
+        address: {{ .ZitiEdgeCtrlAPI }}
     # identity - optional
-    # Allows the webListener to have a specific identity instead of defaulting to the root `identity` section.
-    #    identity:
-    #      cert:                 ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-client.cert.pem
-    #      server_cert:          ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-server.cert.pem
-    #      key:                  ${ZITI_SOURCE}/ziti/etc/ca/intermediate/private/ctrl.key.pem
-    #      ca:                   ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ca-chain.cert.pem
+    # Allows the webListener to have a specific identity instead of defaulting to the root 'identity' section.
+    identity:
+      ca:          "{{ .ZitiPKI }}/{{ .ZitiEdgeCtrlIntermediateName }}/certs/{{ .ZitiEdgeCtrlIntermediateName }}.cert"
```

Comment:
i had to cat the signing cert onto this cert chain. we need to test this and make sure it works. see: https://github.com/openziti/ziti/pull/560/files